tests/test_fastapi.sh

```diff
@@ -11,7 +11,7 @@ git fetch --tags
 
 pip install -r requirements.txt
 # Install the version of pydantic from the current branch, not the released version used by fastapi
-pip uninstall -y pydantic
+pip uninstall -y pydantic pydantic-core # TODO: pydantic-core is temporal here
 cd .. && pip install . && cd fastapi
```

Comment:
Had to add `pydantic-core`, I think this has to be here in order to make pydantic-core installation from branch to work --- Do I keep it here?